src/ringct/rctSigs.cpp

```diff
@@ -765,8 +786,9 @@ namespace rct {
 
           return true;
         }
-        catch(...)
+        catch (const std::exception &e)
         {
+          LOG_PRINT_L1("Error in verRct: " << e.what());
           return false;
         }
```

Comment:
There should not be anything throwing ints, strings, etc. But I'll add a ... anyway.